src/Tasks/Culture.cs

```diff
@@ -29,7 +30,8 @@ internal struct ItemCultureInfo
         internal static ItemCultureInfo GetItemCultureInfo
         (
             string name,
-            string dependentUponFilename
+            string dependentUponFilename,
+            ITaskItem item = null
```

Comment:
Why pass the whole item when a bool could suffice?

Reply:
I considered this, and passing the item doesn't do much harm, but it also doesn't make much sense to pass it along when only 1 aspect of it will be checked. Reverting to passing a boolean.